draft-ietf-scitt-architecture.md

```diff
@@ -367,43 +367,48 @@ This section describes at a high level, the three main roles and associated proc
 
 Transparency Services MAY support additional APIs for auditing, for instance, to query the history of Signed Statements.
 
-Typically a Transparency Services has a single Issuer identity which is present in the `iss` claim of Receipts for that service.
+Typically a Transparency Service has a single Issuer identity which is present in the `iss` claim of Receipts for that service.
 
 Multi-tenant support can be enabled through the use of identifiers in the `iss` claim, for example, `ts.example` may have a distinct Issuer identity for each sub domain, such as `customer1.ts.example` and `customer2.ts.example`.
 
-### Initialization {#ts-initialization}
+### Registration Policies
+
+Registration Policies refer to additional checks over and above the Mandatory Registration Checks that are performed before a Signed Statement is accepted to be registered to the Append-only Log.
+
+Transparency Services MUST maintain Registration Policies which govern whether or not a given Signed Statement is eligible for registration.
+
+Registration Policies MUST be made transparent and available to all clients of the Transparency Service by registering them as Signed Statements on the Append-only Log.
 
-The Append-only Log is empty when the Transparency Service is initialized.
-The first entry that is added to the Append-only Log MUST be a Signed Statement including key material.
-The second set of entries are Signed Statements for additional domain-specific Registration Policy.
-The third set of entries are Signed Statements for Artifacts.
-From here on a Transparency Service can check Signed Statements on registration via policy (that is at minimum, key material and typically a Registration Policy) and is therefore in a reliable state to register Signed Statements about Artifacts or a new Registration Policy.
 
-### Registration Policies
 
-Registration Policies refer to the checks that are performed before a Signed Statement is registered to an Append-only Log, and a corresponding Receipt becomes available.
+This specification leaves implementation, encoding and documentation of Registration Policies to the operator of the Transparency Service.
+
+#### Mandatory Registration Checks
+
+Transparency Services MUST, at a minimum, perform the following checks before registering a Signed Statement:
+* Authenticate the Issuer of the Signed Statement
 
-As a minimum, a Transparency Service MUST authenticate the Issuer of Signed Statements, which requires a trust anchor in the form of an already registered Signed Statement including key material (see {{ts-initialization}}).
-As defined in {{RFC6024}}, "A trust anchor represents an authoritative entity via a public key and associated data.
+To authenticate the Issuer of the Signed Statement the Transparency Service MUST refer to a trust anchor as defined in {{RFC6024}}: "A trust anchor represents an authoritative entity via a public key and associated data.
```

Comment:
```suggestion
The Transparency Service MUST authenticate the Issuer of Signed Statements by validating the COSE signature and checking the identity of the issuer through one of its configured trust anchors, using the `x5t` and `kid` headers in the protected header as hints. For instance, for X.509 signed claims the Transparency Service must validate a complete certificate chain from the certificate identified by `x5t` to one of the trusted root authority certificate of the Transparency Service.
```

Reply:
yes

Reply:
I will give it a 👍 but the use of the word is traipsing into configuration/policy territory that we have been trying to avoid.